File: app/src/main/java/com/smile/karaokeplayer/Constants/PlayerConstants.java
package com.smile.karaokeplayer.Constants;

public final class PlayerConstants {
    private PlayerConstants() {
    }

    public static final String LOG_TAG = "MediaSessionCompatTag";
    public static final String PlayingParamOrigin = "PlayingParamOrigin";
    public static final String PlayingParamState = "PlayingParam";
    public static final String TrackSelectorParametersState = "TrackSelectorParameters";
    public static final String NumberOfVideoTracksState = "NumberOfVideoTracks";
    public static final String NumberOfAudioTracksState = "NumberOfAudioTracks";
    public static final String OrderedSongListState = "OrderedSongList";
    public static final String MediaUriState = "MediaUri";
    public static final String CanShowNotSupportedFormatState = "CanShowNotSupportedFormat";
    public static final String VideoTrackIndicesListState = "VideoTrackIndicesList";
    public static final String AudioTrackIndicesListState = "AudioTrackIndicesList";

    public static final int PrivacyPolicyActivityRequestCode = 10;
    public static final int FILE_READ_REQUEST_CODE = 1;
    public static final int SONG_LIST_ACTIVITY_CODE = 2;
    public static final int PlayerView_Timeout = 10000;  //  10 seconds
    public static final int NoVideoTrack = -1;
    public static final int NoAudioTrack = -1;
    public static final int NoAudioChannel = -1;
    public static final int MaxProgress = 100;
    public static final int PlayingMusic = 1;
    public static final int PlayingVocal = 2;
    public static final int NoRepeatPlaying = 0;
    public static final int RepeatOneSong = 1;
    public static final int RepeatAllSongs = 2;

    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
    public static final String IsPlaySingleSongState = "IsPlaySingleSong";
    public static final String SongInfoState = "SingleSongInfo";
    public static final String PlayerBaseActivityIntent = "PlayerBaseActivityIntent";
}

File: app/src/main/java/com/smile/karaokeplayer/constants/PlayerConstants.kt
package com.smile.karaokeplayer.constants

object PlayerConstants {
    const val LOG_TAG = "MediaSessionCompatTag"
    const val PlayingParamOrigin = "PlayingParamOrigin"
    const val PlayingParamState = "PlayingParam"
    const val TrackSelectorParametersState = "TrackSelectorParameters"
    const val NumberOfVideoTracksState = "NumberOfVideoTracks"
    const val NumberOfAudioTracksState = "NumberOfAudioTracks"
    const val OrderedSongListState = "OrderedSongList"
    const val MediaUriState = "MediaUri"
    const val CanShowNotSupportedFormatState = "CanShowNotSupportedFormat"
    const val AudioTrackIndicesListState = "AudioTrackIndicesList"
    const val PrivacyPolicyActivityRequestCode = 10
    const val PlayerView_Timeout = 10000 //  10 seconds
    const val NoAudioTrack = -1
    const val NoAudioChannel = -1
    const val MaxProgress = 100
    const val NoRepeatPlaying = 0 // Player.REPEAT_MODE_OFF
    const val RepeatOneSong = 1 // Player.REPEAT_MODE_ONE
    const val RepeatAllSongs = 2 // Player.REPEAT_MODE_ALL

    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
    const val IsPlaySingleSongState = "IsPlaySingleSong"
    const val SongInfoState = "SingleSongInfo"

    const val Uri_List: String = "Uri_List"
}